src/randgraphs.jl

```diff
@@ -96,11 +96,10 @@ end
 
 function _suitable(edges::Set{Edge}, potential_edges::Dict{Int, Int})
     isempty(potential_edges) && return true
-    for (s1, s2) in combinations(collect(keys(potential_edges)), 2)
-        if (s1 > s2)
-            s1, s2 = s2, s1
-        end
-        ∉(Edge(s1, s2), edges) && return true
+    list = keys(potential_edges)
+    for s1 in list, s2 in list
```

Comment:
It might make more sense to sort the list, so you can then do

```
list = sort(keys(potential_edges))
for is1 = 1:length(list), is2 = is1:length(list)
  s1 = list[is1], s2 = list[is2]
...
```

Comments on relative efficiency?